web/crux/crux-docker-entrypoint.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
-export NODE_ENV=production
```

Comment:
Because it will override the variable coming from .env or somewhere else. It will cause issues when you run crux from docker image, and the spawned dagent won't be in debug so it will refuse operations if the GRPC connection is insecure. Basically it sabotages non-prod usage.

Reply:
is see. we could use it like this:
If it's not defined, we set it to production as a default in the entrypoint otherwise, we use the provided value

Reply:
It is the default, it's defined in the Dockerfile. Thing is we can override it when it's in the Dockerfile, but if we define it inside, then it will override our ENVs.